addons/website/static/src/scss/website.navbar.mobile.scss

```diff
@@ -16,13 +16,17 @@
         $-font-size-lg: 18px;
         $-entry-hover-bg: rgba(invert($-color), .08);
 
-        // = Invalidate top-menu native 'grid' layout
+
+        // = Invalidate top-menu native 'flex' layout
         // --------------------------------------------------------------------
-        &, .o_menu_sections li, .dropdown-menu.show {
-            display: block;
+        display: block !important;
+
+        .o_menu_toggle {
+            display: inline !important;
```

Comment:
This commit modifies a file and move it at the same time: never do that, move/rename it first then modify it in another commit. Here Github is smart enough to understand it but that might not be the case of all Git configurations and that would be annoying to rebase / cherry pick.

Reply:
Done thanks